stable/airflow/README.md

```diff
@@ -47,9 +48,6 @@ For steps you must take when upgrading this chart, please review:
 * [v7.1.X → v7.2.0](UPGRADE.md#v71x--v720)
 * [v7.0.X → v7.1.0](UPGRADE.md#v70x--v710)
 * [v6.X.X → v7.0.0](UPGRADE.md#v6xx--v700)
-* [v5.X.X → v6.0.0](UPGRADE.md#v5xx--v600)
```

Comment:
why do you remove these llinks?

Reply:
They are still there, in UPGRADE.md, but I don't want the main page to get too cluttered.

Reply:
ok no pbl